### ISSUE TITLE
Problems writing render extension, set DocumentBuilderFactory in SbgnUtil.addAnnotation namespace aware?

### ISSUE THREAD

**Issue by tczauderna** [sbgn/libsbgn#53]
I have some problems writing the render extension using milestone 3.
I had already a brief discussion with Frank but there was not really a solution.

I think I have found a solution after further investigation.

I got it to work by setting the `DocumentBuilderFactory` namespace aware in `SbgnUtil.addAnnotation`
```
DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
dbf.setNamespaceAware(true);
```

Before creating a pull request, can someone please check if this works for others as well?

**Comment by fbergmann** [comment]
hello tobias, i'll have a look and add some test cases ...

**Comment by fbergmann** [comment]
for me it works with the additional line, so i've committed it. If you have a specific error, we should add it as test case, so we can ensure it works across different java versions.

**Comment by tczauderna** [comment]
Fixed with commit 31244f8
